rest-api-spec/src/main/resources/rest-api-spec/api/security.disable_user_profile.json

```diff
@@ -0,0 +1,41 @@
+{
+  "security.disable_user_profile":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/security-api-disable-user-profile.html",
+      "description":"Disables a user profile so it's not visible in user profile searches."
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_security/profile/{uid}/_disable",
+          "methods":[
+            "PUT",
+            "POST"
+          ],
+          "parts":{
+            "uid":{
+              "type":"string",
+              "description":"Unique identifier for the user profile"
+            }
+          }
+        }
+      ]
+    },
+    "params":{
+      "refresh":{
+        "type":"enum",
+        "options":[
+          "true",
+          "false",
+          "wait_for"
+        ],
+        "description":"If `true` (the default) then refresh the affected shards to make this operation visible to search, if `wait_for` then wait for a refresh to make this operation visible to search, if `false` then do nothing with refreshes."
```

Comment:
Default is `wait_for`.

Reply:
Errr .. thanks for catching it. Updated!